.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+on:
+  push:
+    branches: [ master ]
+    
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  release:
+    name: Release new version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2.1.0
+      - name: Install deps
+        uses: borales/actions-yarn@v2.1.0
+        with:
+          cmd: install --pure-lockfile # will run `yarn install` command
+      - name: Semantic release
+        uses: cycjimmy/semantic-release-action@v2.4.1
+        semantic_version: next
+        extra_plugins: |
```

Comment:
You will need to add the semantic release configuration in the `package.json` for the plugins:

Based on ramda-land, try something like:

```json
{
    "plugins": [
      [
        "@semantic-release/commit-analyzer",
        {
          "preset": "angular",
          "releaseRules": [
            {
              "type": "docs",
              "release": "patch"
            },
            {
              "type": "build",
              "release": "patch"
            }
          ]
        }
      ],
      "@semantic-release/release-notes-generator",
      "@semantic-release/npm",
      "@semantic-release/github",
      [
        "@semantic-release/git",
        {
          "assets": [
            "package.json",
            "docs"
          ],
          "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
        }
      ]
    ]
  }
```

If you want, you can avoid most of the configuration except the git one - this will update the `package.json` in the master branch with the generated semver number, although I don't know if we want to keep that.